Sources/Controllers/Map/OAMapHudViewController.mm

```diff
@@ -973,6 +1055,23 @@ - (BOOL) topControlsVisible
     return _statusBarView.alpha > 0;
 }
 
+- (void) setTopControlsAlpha:(CGFloat)alpha
+{
+    CGFloat alphaEx = self.contextMenuMode ? 0.0 : alpha;
```

Comment:
Will this leave top controls active?

Reply:
https://www.dropbox.com/s/wtmbnp59hsx1qq8/Screen%20Recording%202021-08-05%20at%2015.57.01.mov?dl=0